docs/blog/newsletter-september-2022.rst

```diff
@@ -0,0 +1,103 @@
+
+--------------------
+When to use acronyms
+--------------------
+
+Style guides such as the `Microsoft Style Guide <https://docs.microsoft.com/en-us/style-guide/>`__ suggest spelling out acronyms and abbreviations at the first use, except when the short versions are well known. This led to the question of how do you know that an acronym is known well enough? Where do you draw the line?
+
+People noted that, as with many questions about documentation, it depends on your audience and what they know. Some pointed to acronyms such as URL, HTTP, and API being better known than the spelled-out versions, especially among more technical audiences. They worried the long versions would actually be harder to understand. One person pointed to a threshold of 90% of users knowing the acronym as enough to use it without defining it.
+
+Others worried relying on a rule about common knowledge would mean people would abbreviate too much. They'd assume their audience knew technical terms like they do. And your audience can always expand beyond your initial ideas, so you may have more nontechnical people reading than you might think.
+
+Many people agreed that it's better to err on the side of defining too many acronyms, rather than too few. You never know if someone will see the term for the first time in your docs. It can be good to use some acronyms to shorten text when they appear multiple times, but take care not to let your docs get too full of acronyms that they become incomprehensible.
+
+------------------------------------------------
+Improving communication about what's in releases
+------------------------------------------------
+
+We recently had a discussion about how to keep communication in sync between engineering, product managers (PMs), and documentation. In particular, about making sure that documentarians know if the product is changing so they know what needs doing for a release.
+
+The original poster hadn't had much success asking engineering to keep them informed -- often engineering would forget. So they were thinking of trying to improve this with better process, via PMs; surely PMs keep tabs on what's changing in the product and are aware of what changes might need doc work?
+
+The answer is, it depends, because how PMs work with engineering can vary hugely among organizations and even among teams. Some PMs are the main point of contact for what's changing, and want to be kept in the loop. Others are focused on handling user requests and reviewing feature designs and aren't really involved in launching, in which case so engineering is a better source.
```

Comment:
```suggestion
The answer is, it depends, because how PMs work with engineering can vary hugely among organizations and even among teams. Some PMs are the main point of contact for what's changing, and want to be kept in the loop. Others are focused on handling user requests and reviewing feature designs and aren't really involved in launching, in which case engineering is a better source.
```